ChangeLog.md

```diff
@@ -1,7 +1,12 @@
-# DGtal 1.2 (dev)
+# DGtal 1.2
 
 ## New Features / Critical Changes
 
+- *New Feature*
+  - DGtal now has a python binding `pip install dgtal`! For all
+    details on the list of classes available in python, you can have a
+    loo to: Pablo Hernandez-Cerdan [#1543](https://github.com/DGtal-team/DGtal/pull/1528) 
```

Comment:
loo -> look

I think is good to link to the PR, but there is also a README, with the same information, in the `wrap` folder here: https://github.com/DGtal-team/DGtal/tree/master/wrap

Reply:
thanks for the feedback